src/ui.form-select/USelect.vue

```diff
@@ -599,34 +106,33 @@ const isTop = computed(() => {
   return preferredOpenDirection.value === DIRECTION.top;
 });
 
-const elementId = props.id || useId();
-
-const i18nGlobal = tm(USelect);
-const currentLocale = computed(() => merge(defaultConfig.i18n, i18nGlobal, props.config.i18n));
-
 const iconSize = computed(() => {
   const sizes = {
     sm: "xs",
     md: "sm",
     lg: "md",
   };
 
-  return sizes[props.size];
+  return sizes[props.size] as IconSize;
 });
 
 const inputPlaceholder = computed(() => {
   const message = currentLocale.value.addMore;
 
-  return props.multiple && localValue.value.length ? message : props.placeholder;
+  return props.multiple && localValue.value?.length ? message : props.placeholder;
 });
 
 const dropdownValue = computed({
   get: () => props.modelValue,
   set: (newValue) => {
-    let value = newValue;
+    let value;
 
     if (props.multiple) {
-      value = Array.isArray(props.modelValue) ? [...props.modelValue, newValue] : [newValue];
+      const multipleValue = Array.isArray(props.modelValue) ? props.modelValue : [];
```

Comment:
Wasn't it much cleaner before? This line look useful.

Reply:
we need to apply `flat()` and by this way it looks more readable.

You can suggest resulting piece of code in comments below.